lms/djangoapps/branding/__init__.py

```diff
@@ -27,25 +27,26 @@ def get_visible_courses(org=None, filter_=None):
         filter_ (dict): Optional parameter that allows custom filtering by
             fields on the course.
     """
-    current_site_org = configuration_helpers.get_value('course_org_filter')
+    courses = []
+    current_site_orgs = configuration_helpers.get_current_site_orgs()
 
-    if org and current_site_org:
+    if org and current_site_orgs:
         # Return an empty result if the org passed by the caller does not match the designated site org.
         courses = CourseOverview.get_all_courses(
-            org=org,
+            orgs=[org],
             filter_=filter_,
-        ) if org == current_site_org else []
+        ) if org in current_site_orgs else []
```

Comment:
also I'd rather see a nested if, versus this format -- I'm not sure if there's a big difference, though -- ie:

```
if org and current_site_orgs:
    if org in current_site_orgs:
        courses = CourseOverview()
elif org:
elif current_site_orgs:
else:

Also for all of these conditions it looks like we're just trying to figure out what to use for the `orgs` value when we call `CourseOverview.get_all_courses()` -- can we move that out of the conditional block and just focus on determining the orgs value with all the if/elif/else stuff?
```